accesspoc/dips/helpers.py

```diff
@@ -36,3 +36,14 @@ def update_instance_from_dict(instance, dict):
             continue
         setattr(instance, field, value)
     return instance
+
+
+def str_to_bool(value):
```

Comment:
See my comment above, you wouldn't need this if the values you put in the database are all encoded using JSON.